### DISCUSSION TITLE
Different Audio

### DISCUSSION THREAD

**Discussion by j33l** [community/events#35] [Community]
in this "How to open source as a startup" event, I'm getting audio from the Keynote event...

**Comment by Justintime50** [comment]
Same, it's the audio from `TrackOne` and needs to be corrected to `TrackFive`

**Reply by ahjota** [reply]
`TrackFive` is the problem stream.

**Comment by queenofcorgis** [comment]
Sharing this feedback and your experience with our internal team - thank you for reporting! Please note all of our core content will be broadcast live and posted on-demand on githubuniverse.com for anyone to watch after the event. Presentations will be available in this repo, post-event.
